src/components/mobile/CoursesMob.js

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  TiSocialFacebook,
+  TiSocialTwitter,
+  TiSocialInstagram,
+} from 'react-icons/ti';
+import { getList } from '../../redux/features';
+import cockpit from '../../images/cockpit.png';
+import technic from '../../images/technic.png';
+import cabin from '../../images/cabin.png';
+import plane from '../../images/plane.png';
+
+const CoursesMob = () => {
+  const state = useSelector((state) => state.featuresReducers);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getList);
+  }, []);
+
+  const imageDisplay = (item) => {
+    if (item.course_type === 'Cockpit Course') {
+      return (
+        <img
+          className="w-[200px] mx-auto bg-slate-200 rounded-full"
+          src={cockpit}
+          alt="/"
+        />
+      );
+    }
+    if (item.course_type === 'Cabin Course') {
+      return (
+        <img
+          className="w-[200px] mx-auto bg-cyan-100 rounded-full"
+          src={cabin}
+          alt="/"
```

Comment:
[OPTIONAL] Maybe this img can be a reusable component: You judge for yourself

Reply:
Thank you. Refactored both desktop & mobile components.